### PR TITLE
[FAQ] how to change coordinator hidden service

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This adds a quick FAQ about how to change the coordinator onion.

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -1408,6 +1408,17 @@ Please note that Wasabi is designed for the dark theme, and some color schemes m
 ![Wasabi Wallet white theme](/WhiteMode.png "Wasabi Wallet white theme")
 :::
 
+:::details
+### How can I connect to a different coordinator?
+
+To reach the coordinator you need their hidden service .onion address.
+Open the `Conf.json`  file through the top menu bar.
```
Comment (RiccardoMasutti) [inline docs/FAQ/FAQ-UseWasabi.md:1415]:
```suggestion
Open the `Conf.json` file through the top menu bar.
```

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -1408,6 +1408,17 @@ Please note that Wasabi is designed for the dark theme, and some color schemes m
 ![Wasabi Wallet white theme](/WhiteMode.png "Wasabi Wallet white theme")
 :::
 
+:::details
+### How can I connect to a different coordinator?
+
+To reach the coordinator you need their hidden service .onion address.
+Open the `Conf.json` file through the top menu bar.
```
Comment (DanGould) [inline docs/FAQ/FAQ-UseWasabi.md:1415]: change line to 2. From menu: File/Open/Config File.

`Conf.json` -> `Config.json`

Comment (MaxHillebrand) [reply docs/FAQ/FAQ-UseWasabi.md:1415]:
```suggestion
Open the `Config.json` file through the top menu bar.
```

**Comment by yahiheb** [comment]
Why should we add this to the docs?

**Comment by DanGould** [comment]
@yahiheb because it answers a question that is asked frequently

**Comment by MaxHillebrand** [comment]
@yahiheb because Wasabi =/ zkSNACKs.

although this repository ships a package that by default connects to the zkSNACKs coordinator, there are already and will continue to be other coordinators operated. That's a feature, not a bug!!

 Users can and will connect to different coordinators, so it ought to be explained how to do it properly, so to not mess up.

**Comment by RiccardoMasutti** [comment]
> @yahiheb because Wasabi =/ zkSNACKs.
> 
> although this repository ships a package that by default connects to the zkSNACKs coordinator, there are already and will continue to be other coordinators operated. That's a feature, not a bug!!
> 
> Users can and will connect to different coordinators, so it ought to be explained how to do it properly, so to not mess up.

I agree on explaining how to change the coordinator variable, but I believe that the lack of liquidity in the current other coordinators can be a privacy issue and I would advise against most users to change it.

**Comment by MaxHillebrand** [comment]
yes Riccardo, this is a concern, and notice that's the reason why I do not link to the onion of a different coordinator, as this would be a recommendation, and thus carefully ought to be considered.

Nevertheless, changing the coordinator onion, even if it's for your own setup on regtest, is useful information worthy to be in the docs.

**Comment by yahiheb** [comment]
Yes Wasabi =! zkSNACKs, but the docs are about Wasabi stuff mainly, so why should we add anything not really related to Wasabi?
I mean does Wasabi have any other coordinator that users should manually connect to? AFAIK no, so there is no need for this addition.

> @yahiheb because it answers a question that is asked frequently

But that question is not related to Wasabi.
Don't get me wrong I am all in for chaincase but I don't think we should add this here just because people want to use Wasabi to connect to chaincase's coordinator.

**Comment by MaxHillebrand** [comment]
> why should we add anything not really related to Wasabi?

Because changing the coordinator is a feature in the Wasabi client.

This is not the zkSNACKs documentation [although it's of course in the zkSNACKs repo, lol], but the Wasabi documentation. And the Wasabi client, is designed in such a way to connect to different than the default coordinator.

imho such a core architecture feature ought to be documented, specifically when there are real use cases for changing it.

> I mean does Wasabi have any other coordinator that users should manually connect to?

Yes, for example the mainchain coordinator of chaincase, and I know several users who connect to it with their Wasabi desktop client.

I think it's a reasonable assumption to see in the future new coordinators with unique round parameters.

**Comment by RiccardoMasutti** [comment]
@MaxHillebrand @yahiheb What if we add a `:::warning` explaining that by changing the default coordinator, the user expose himself to a series of problems not related to the default coordinator? (eg. low liquidity, less privacy/anonset per round, etc.)

**Pull request closed without merging**